elyra/pipeline/tests/test_pipeline_processor_local.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2018-2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from elyra.pipeline import PipelineParser
+from elyra.pipeline.processor_local import LocalPipelineProcessor
+from .util import _read_pipeline_resource
+
+
+def test_pipeline_execution_order():
+    expected_operation_names = ['a', 'b', 'c', 'd', 'e', 'f', 'x', 'y', 'g', 'h']
```

Comment:
As noted on the test pipeline, this is essentially the same order the file is in already.  Please mixup the nodes in the file.

Reply:
I still think the order of the operations need to change within the pipeline file.  Right now, you could not perform any sorting and get this same answer.  You will likely need to change the order of the expected names if this is the way you're going to validate.